index.bs

```diff
@@ -3460,7 +3457,8 @@ interfaces that have interface members with these names.
 
         typeof SessionManager.prototype.values;            // Evaluates to "function"
         var it = sm.values();                              // values() returns an iterator object
-        String(it);                                        // Evaluates to "[object SessionManager Iterator]"
+<!--    String(it);                                        // Evaluates to "[object SessionManagerIterator]"
+                                                           // TODO: https://github.com/heycam/webidl/issues/419 -->
         typeof it.next;                                    // Evaluates to "function"
 
         // This loop will alert "anna" and then "brian".
```

Comment:
Sorry I didn't notice that before, can you change this to "log" stuff rather than "alert" it?

So `s/This loop will also alert/This loop will also log/` and `s/window.alert/console.log/`.